saleor/core/search_tasks.py

```diff
@@ -13,94 +15,101 @@
 
 task_logger = get_task_logger(__name__)
 
-BATCH_SIZE = 10000
+BATCH_SIZE = 1000
 
 
 @app.task
-def set_user_search_document_values(total_count, updated_count):
-    qs = User.objects.filter(search_document="").prefetch_related("addresses")[
-        :BATCH_SIZE
-    ]
+def set_user_search_document_values(updated_count: int = 0) -> None:
+    qs = list(
```

Comment:
I think we should rename qs to something different as it's not queryset anymore after list casting. I was a little bit confused few lines below when I saw if len(qs) < BATCH_SIZE:. I wanted to write that we should use .count() instead but then I saw it's not a queryset :)

Reply:
I'll rename it but I'd also never use `.count()` on a QuerySet that is going to be evaluated as it would hit the database with an additional `COUNT()` query.